resources/client_install.rb

```diff
@@ -17,22 +17,11 @@
 #
 
 property :version, String, default: '9.6'
-property :enable_pgdg, [true, false], default: true
-property :enable_pgdg_source, [true, false], default: false
-property :enable_pgdg_updates_testing, [true, false], default: false
-property :enable_pgdg_source_updates_testing, [true, false], default: false
-property :yum_gpg_key_uri, [String, nil], default: nil
-property :apt_gpg_key_uri, [String, nil], default: nil
+property :auto_setup_repo, [true, false], default: true
```

Comment:
does "setup_repo" sound better [bike-shed-emoji] ?

Reply:
I approve of that bike shed color

Reply:
If you have this on false, but also provide true to the server_install resource?